tapdb/sqlc/migrations/000020_asset_unique_key.up.sql

```diff
@@ -0,0 +1,4 @@
+CREATE UNIQUE INDEX assets_genesis_id_script_key_id_anchor_utxo_id_unique
```

Comment:
Shouldn't we also drop the constraint `UNIQUE(asset_id, genesis_id, script_key_id)` ? Because that constraint makes no sense, and is still there: https://github.com/lightninglabs/taproot-assets/blob/dd8ba788757bd7d1b7c16ab0b97921283d4d0f5e/tapdb/sqlc/migrations/000002_assets.up.sql#L221

Reply:
Unfortunately we can't easily remove that constraint, because it is not a named one (we should really avoid this in the future, exactly due to this reason).
The only way to get rid of this would be to create a completely new temporary asset table (without the constraint), copy the data, disable all foreign keys, drop the old table, rename the temporary table to the previous name, enable all foreign keys again.
We should probably do that at some point, but don't want to write such a script under time pressure.

Reply:
Made an issue for this: #967